web/wp-content/themes/dctx-v2/inc/template-tags.php

```diff
@@ -0,0 +1,540 @@
+		$time_string = '<time class="entry-date published updated" datetime="%1$s">%2$s</time>';
+		if ( get_the_time( 'U' ) !== get_the_modified_time( 'U' ) ) {
+			$time_string = '<time class="entry-date published" datetime="%1$s">%2$s</time><time class="updated" datetime="%3$s">%4$s</time>';
+		}
+
+		$time_string = sprintf(
+			$time_string,
+			esc_attr( get_the_date( 'c' ) ),
+			esc_html( get_the_date() ),
+			esc_attr( get_the_modified_date( 'c' ) ),
+			esc_html( get_the_modified_date() )
+		);
+
+		$posted_on = sprintf(
+			/* translators: the date the post was published */
+			esc_html_x( 'Posted on %s', 'post date', 'dctx' ),
+			'<a href="' . esc_url( get_permalink() ) . '" rel="bookmark">' . $time_string . '</a>'
+		);
+
+		$byline = sprintf(
+			/* translators: the post author */
+			esc_html_x( 'by %s', 'post author', 'dctx' ),
+			'<span class="author vcard"><a class="url fn n" href="' . esc_url( get_author_posts_url( get_the_author_meta( 'ID' ) ) ) . '">' . esc_html( get_the_author() ) . '</a></span>'
+		);
+
+		echo '<span class="posted-on">' . $posted_on . '</span><span class="byline"> ' . $byline . '</span>'; // WPCS: XSS OK.
+
+	}
+endif;
+
+if ( ! function_exists( 'dctx_entry_footer' ) ) :
+	/**
+	 * Prints HTML with meta information for the categories, tags and comments.
+	 *
+	 * @author WDS
+	 */
+	function dctx_entry_footer() {
+		// Hide category and tag text for pages.
+		if ( 'post' === get_post_type() ) {
+			/* translators: used between list items, there is a space after the comma */
+			$categories_list = get_the_category_list( esc_html__( ', ', 'dctx' ) );
+			if ( $categories_list && dctx_categorized_blog() ) {
+				/* translators: the post category */
+				printf( '<span class="cat-links">' . esc_html__( 'Posted in %1$s', 'dctx' ) . '</span>', $categories_list ); // WPCS: XSS OK.
+			}
+
+			/* translators: used between list items, there is a space after the comma */
+			$tags_list = get_the_tag_list( '', esc_html__( ', ', 'dctx' ) );
+			if ( $tags_list ) {
+				/* translators: the post tags */
+				printf( '<span class="tags-links">' . esc_html__( 'Tagged %1$s', 'dctx' ) . '</span>', $tags_list ); // WPCS: XSS OK.
+			}
+		}
+
+		if ( ! is_single() && ! post_password_required() && ( comments_open() || get_comments_number() ) ) {
+			echo '<span class="comments-link">';
+			comments_popup_link( esc_html__( 'Leave a comment', 'dctx' ), esc_html__( '1 Comment', 'dctx' ), esc_html__( '% Comments', 'dctx' ) );
+			echo '</span>';
+		}
+
+		edit_post_link(
+			sprintf(
+				/* translators: %s: Name of current post */
+				esc_html__( 'Edit %s', 'dctx' ),
+				the_title( '<span class="screen-reader-text">"', '"</span>', false )
+			),
+			'<span class="edit-link">',
+			'</span>'
+		);
+	}
+endif;
+
+/**
+ * Display SVG Markup.
+ *
+ * @param array $args The parameters needed to get the SVG.
+ *
+ * @author WDS
+ */
+function dctx_display_svg( $args = array() ) {
+	echo dctx_get_svg( $args ); // WPCS XSS Ok.
```

Comment:
:warning: **Warning**: Using the WPCS native whitelist comments is deprecated. Please use the PHPCS native &quot;phpcs:ignore Standard.Category.SniffName.ErrorCode&quot; annotations instead. Found: // WPCS XSS Ok.\n (*WordPress.Security.EscapeOutput.DeprecatedWhitelistCommentFound*).